.travis.yml

```diff
@@ -8,27 +8,17 @@ before_install:
 - ./miniconda.sh -b -p ./lib/miniconda
 - export PATH=$(pwd)/lib/miniconda/bin:$PATH
 - conda update --yes conda
+- wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-113.0.0-linux-x86_64.tar.gz -nv
+- tar zxvf google-cloud-sdk-113.0.0-linux-x86_64.tar.gz -C lib
+- export PATH=$(pwd)/lib/google-cloud-sdk/bin:$PATH
 install:
 - conda install --yes python=$TRAVIS_PYTHON_VERSION cryptography numpy scipy
-- pip install -r requirements.txt
-- pip install -r test_requirements.txt
-- pip install -e src/common -e src/scraper -e src/server -e src/topicmodeller -e src/learner -e src/orchestrator
+- tools/install_envs.sh
 - pip install coveralls
-- python -m nltk.downloader stopwords
-- python -m nltk.downloader punkt
-before_script:
-- wget http://storage.googleapis.com/appengine-sdks/featured/google_appengine_1.9.28.zip -nv
-- unzip -q google_appengine_1.9.28.zip -d lib
-# symblic link between the 2 locations where "google" is a root of python packages (to make 'import google.*' work)
-- export GOOGLE_PIP_DIR=$(pip show protobuf | grep Location | awk {'print $2'})/google
-- ln -s $(pwd)/lib/google_appengine/google/appengine $GOOGLE_PIP_DIR/appengine
-- ln -s $(pwd)/lib/google_appengine/google/net $GOOGLE_PIP_DIR/net
-- ./tools/start_gae.sh &
-- sleep 3
 script:
-- py.test src
-- coverage run --source=src --omit=*tests* -m py.test src
-- ./tools/run_pylint.py src/
+- ./tools/start_tests.sh
+- python tools/run_pylint.py src
```

Comment:
Because travis is already on its "local env", its local env plays in travis the same role than "global_env" in dev mode.
travis call run_pylint.py directly, I only use run_pylint.sh in my dev setup (see the extension).

the script in .sh is needed because pycharm don't reuse the virtualenv when running an external tool